kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -675,7 +675,7 @@ def foundry_list(foundry_root: Path) -> list[str]:
         proof_digest = foundry.proof_digest(contract_name, test_name)
         if APRProof.proof_exists(proof_digest, apr_proofs_dir):
             apr_proof = APRProof.read_proof(proof_digest, apr_proofs_dir)
-            lines.extend(apr_proof.summary)
+            lines.extend(str(apr_proof.summary))
```

Comment:
```suggestion
            lines.extend(apr_proof.summary.lines)
```

We probably shouldn't be joining them by default.